UIKitHometasks/UIKitHometasks/SongInformationViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  SongInformationViewController.swift
+//  UIKitHometasks
+//
+//  Created by Валентин Коскин on 26.09.2022.
+//
+
+import AVFoundation
+import UIKit
+/// viewController описывает экран с информаицей о песне
+class SongInformationViewController: UIViewController {
```

Comment:
```suggestion
final class SongInformationViewController: UIViewController {
```
final - ставим перед всеми классами для производительности и ускорения горячего билда.